samples/core/exit_handler/exit_handler.py

```diff
@@ -21,7 +21,7 @@
 def gcs_download_op(url):
     return dsl.ContainerOp(
         name='GCS - Download',
-        image='google/cloud-sdk:216.0.0',
+        image='google/cloud-sdk:latest',
```

Comment:
Is this necessary? If it need a later version, can we ping to that version instead of using latest?

Reply:
I can verify again, that was my guess, but it seemed we may not need this change.

Reply:
I pinned to latest version and verified this is required. Using old cloud-sdk makes tests flaky.